### ISSUE TITLE
Improve battery life

### ISSUE THREAD

**Issue by ara4n** [vector-im/vector-android#616]
We need to get a better transport implemented, especially for Fdroid, that minimises bandwidth and syncs minimal notifs rather than doing full /sync

**Comment by VasoVV** [comment]
I don't think that is "enhancement". Almost a half of battery consumption -- it's a "critical" issue!
![Battery life](https://cloud.githubusercontent.com/assets/8181361/19882634/cc5504bc-a017-11e6-8c31-dc80cf6175f3.png)

**Comment by Matrixcoffee** [comment]
In advance of implementing a better transport, perhaps the defaults for background sync could be made less aggressive.

**Comment by tuxayo** [comment]
Conversations and Telegram are battery efficient and don't rely on GCM/push notifications.

They could be sources of inspiration for Riot if the relevant source code can be easily understood or if the devs are reachable for questions about how they managed to do this.

https://github.com/siacs/Conversations
https://github.com/DrKLO/Telegram

**Comment by Mikaela** [comment]
Google Play Version: before disabling background sync and losing notifications making the app useless for me https://github.com/vector-im/riot-android/issues/995 it was using ~20% of battery and Android was daily giving me warnings about high power consumption.

The other battery hungry apps included WhatsApp by 1% and Telegram (and Conversations) which I might use more than Riot weren't even listed.

**Comment by grothesque** [comment]
The following article discusses how the Conversations client manages to use little battery power while not requiring any external "push services":
https://gultsch.de/xmpp_2016.html (see under the heading "An Excuse to Push").
I do not know whether doing anything like this for matrix would require modifying the matrix protocol.

**Comment by santicalcagno** [comment]
See also: [How Tutanota replaced Google’s FCM with their own notification system](https://f-droid.org/en/2018/09/03/replacing-gcm-in-tutanota.html)

**Comment by non-Jedi** [comment]
I'm seeing some pretty significant battery drain from Riot lately, and I'm using the google play version. Not sure if this is a new thing or just something I'm only noticing lately, but it's actually affecting my ability to not have to recharge my phone midday.

**Comment by tuxayo** [comment]
@non-Jedi Is this correlated to an update?

**Comment by menturion** [comment]
Same here.
The Android phone is getting hot with high battery consumption.
